code/api/src/setup/database.js

```diff
@@ -18,13 +18,13 @@ const connection = new Sequelize(databaseConfigEnv.database, databaseConfigEnv.u
 // Test connection
 console.info('SETUP - Connecting database...')
 
-connection
-  .authenticate()
-  .then(() => {
-    console.info('INFO - Database connected.')
-  })
-  .catch(err => {
-    console.error('ERROR - Unable to connect to the database:', err)
-  })
+// connection
```

Comment:
This is the same code that was giving Megan issues during her testing lesson walk-through. It's easiest to comment it out.